example/cra-provider/package.json

```diff
@@ -20,7 +23,7 @@
     "web-vitals": "^2.1.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "GENERATE_SOURCEMAP=false react-scripts start",
```

Comment:
Without this, a bunch of warnings are thrown at server start because of the wallet-connect library